api/serverpb/json/serverpb.json

```diff
@@ -1081,6 +1081,126 @@
         }
       }
     },
+    "/v1/Settings/TestEmailAlertingSettings": {
+      "post": {
+        "tags": [
+          "Server"
+        ],
+        "summary": "TestEmailAlertingSettings tests email alerting SMTP settings by sending testing email.",
+        "operationId": "TestEmailAlertingSettings",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "email_to": {
+                  "description": "Target email address to send the email to.",
+                  "type": "string",
+                  "x-order": 1
+                },
+                "settings": {
```

Comment:
can we make it `email_alerting_settings `, to be consistent with the other endpoints?

Reply:
done